lib/wasm/WasmMainTemplatePlugin.js

```diff
@@ -272,7 +272,17 @@ class WasmMainTemplatePlugin {
 										Template.indent([
 											`promise = WebAssembly.instantiateStreaming(req, ${createImportObject(
 												"importObject"
-											)});`
+											)}).catch(function(reason) {`,
```

Comment:
Could we still log the error here? Or a warning and mentioning that we fallbacked.

Reply:
Is there a way to detect that the error is a mime-type issue? If there's any other reason, it will try to fetch the binary a second time.